src/lib.rs

```diff
@@ -151,16 +151,18 @@ extern crate derive_more;
 
 pub mod bench_utils;
 pub(crate) mod circuit;
+pub mod config;
 pub mod constants;
 pub mod errors;
 pub mod freeze;
 pub mod keys;
 pub mod mint;
 pub mod parameters;
+pub mod prelude;
 pub mod proof;
 pub mod structs;
 pub mod transfer;
-mod types;
+// mod types;
```

Comment:
Remove this line?

Reply:
yes, I will, btw this PR is far from being finished... There's still 200+ compiler error. I haven't removed this types file just for reference during my refactoring, will remove later.